mlflow/gateway/providers/anthropic.py

```diff
@@ -0,0 +1,104 @@
+from typing import Dict, Any
+
+from fastapi import HTTPException
+from fastapi.encoders import jsonable_encoder
+
+from .base import BaseProvider
+from ..schemas import completions, chat, embeddings
+
+
+class AnthropicProvider(BaseProvider):
+    NAME = "anthropic"
+    SUPPORTED_ROUTES = "completions"
+
+    async def _request(self, path: str, payload: Dict[str, Any]):
+        import aiohttp
```

Comment:
Should we include `atiohttp` as a gateway dependency?

Reply:
yes, good idea. I'll make that change.